### ISSUE TITLE
Total number of people with immunity

### ISSUE THREAD

**Issue by mmarx83** [okestonia/koroonakaart#262]
Hi!

we have been told daily how many vaccinated persons there are. But shouldn't we also add infected people to vaccinated people, so we could get total number of virus/vaccine affected people. Very roughly, today this number is 900 000 (750 000 vaccinated and 150 000 infected). which will make "infection safety" % to be around 67% of whole population, not only crownup's.

**Comment by neuroactive** [comment]
I agree it would be interesting to know the total percentage of the population that has some immunity from the disease. The problem is that some people who recover from Covid go on to get vaccinated, so we can't simply add the number of recovered to the number of vaccinated as those people would be counted twice. I have previously suggested to the authorities who manage the data that it would be useful for them to release data on immunity as they are in the best position to calculate this correctly, but was informed that this information is not likely to be released in the foreseeable future.

**Comment by ostankin** [comment]
Tartu University publishes its monthly estimates of population immunity: https://ut.ee/et/sisu/koroonaviiruse-levimuse-seireuuringu-antikehade-analuusi-tulemused

Not sure if they export any API, but for now this seems the best source of this kind of information.
